tests/jobs/design2params/test_design2params.py

```diff
@@ -186,3 +214,15 @@ def test_open_excel_file_value_missing(input_data):
             design2params._PARAMETERS_TXT,
             log_level=logging.DEBUG,
         )
+
+
+def test_open_excel_file_wrong_defaults(input_data):
+    with pytest.raises(SystemExit):
+        design2params.run(
+            1,
+            "design_matrix_missing_value.xlsx",
```

Comment:
Is it possible to create these files on the fly with `to_excel`? That would make it a bit easier for me to understand what is going on without having to look at the other files in the repo. It would also make it possible to add a unit test to `_read_defaultssheet` above.

Reply:
Yes it is. Will do.

Reply:
Second commit added implementing this.

Reply:
Thanks! Made it much easier to get a grasp of what is going on 👍